### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mikebrow**
Updates to readme per  https://github.com/opencontainers/tob/issues/134#issuecomment-1894346462

### CODE REVIEW

**Review by estesp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ The TOB is responsible for managing conflicts, violations of procedures or guide
...
 * **Vincent Batts [Microsoft]** (term date: 1/29/2023 - 1/29/2025)
-* **Josh Dolitsky [Chainguard]** (term date: 1/29/2022 - 1/29/2024)
-* **Phil Estes [AWS]** (term date: 1/29/2022 - 1/29/2024)
-* **Jon Johnson [Chainguard]** (term date: 1/29/2022 - 1/29/2024)
-* **Nisha Kumar [Oracle]** (term date: 1/29/2022 - 1/29/2024)
-* **Samuel Karp [Google]** (term date: 1/29/2022 - 1/29/2024) [Chair]
 * **Derek McGowan [Docker]** (term date: 1/29/2023 - 1/29/2025)
 * **Aleksa Sarai [SUSE]** (term date: 1/29/2023 - 1/29/2025)
+* **Phil Estes [AWS]** (term date: 1/29/2024 - 1/29/2026)
+* **Samuel Karp [Google]** (term date: 1/29/2024 - 1/29/2026) [Chair]
```
Comment (samuelkarp) [inline README.md:12]: Probably worth removing [Chair] until we have the new chair election.

**Review by samuelkarp — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mikebrow) [reply README.md:12]: on 2/1 maybe?

Comment (mikebrow) [reply README.md]: np.. done :-)

**Review by sudo-bmitch — APPROVED** [review]
LGTM, but I'd hold merging until the 29th.

**Comment by amye** [comment]
Bah, then of course I just smash in all over this. Closing but with several apologies.

**Pull request closed without merging**